lib/policies/read_policy.js

```diff
@@ -77,6 +77,17 @@ class ReadPolicy extends BasePolicy {
      * @since v3.7.0
      */
     this.deserialize = props.deserialize
+
+    /**
+     * Should as_record instance be allocated on the heap before user listener is called in
+     * async commands. If true, the user is responsible for calling as_record_destroy() when done
+     * with the record. If false, as_record_destroy() is automatically called by the client after
+     * the user listener function completes. This field is ignored for sync commands.
```

Comment:
How is this supposed to work from JS-land? JS code can't call the `as_record_destroy` function unless you are going to expose some JS API for that?